content/rs/references/rest-api/quick-start.md

```diff
@@ -0,0 +1,316 @@
+
+## Example requests: cURL
+
+[cURL](https://curl.se/) is a command-line tool that allows you to send HTTP requests from a terminal.
+
+A cURL request includes the following options:
+
+| Option | Description |
+|--------|-------------|
+| -X     | Method (GET, PUT, PATCH, POST, or DELETE) |
+| -H     | Request header, can be specified multiple times |
+| -u     | Username and password information |
+| -d     | JSON data for PUT or POST requests |
+| -F     | Form data for PUT or POST requests, such as for the [`POST /v1/modules`]({{<relref "/rs/references/rest-api/requests/modules/#post-module" >}}) or [`POST /v2/modules`]({{<relref "/rs/references/rest-api/requests/modules/#post-module-v2" >}}) endpoint |
+| -k     | Turn off SSL  verification |
+| -i     | Show headers and status code as well as the response body |
+
+See the [cURL documentation](https://curl.se/docs/) for more info.
+
+### GET request
+
+Use the following cURL command to get a list of databases with the [GET `/v1/bdbs/`]({{<relref "/rs/references/rest-api/requests/bdbs/#get-all-bdbs" >}}) endpoint.
+
+```sh
+$ curl -X GET -H "accept: application/json" \
+              -u "[username]:[password]" \
+              https://[host][:port]/v1/bdbs -k -i
+
+HTTP/1.1 200 OK
+server: envoy
+date: Tue, 14 Jun 2022 19:24:30 GMT
+content-type: application/json
+content-length: 2833
+cluster-state-id: 42
+x-envoy-upstream-service-time: 25
+
+[
+    {
+        ...
+        "name": "tr01",
+        ...
+        "uid": 1,
+        "version": "6.0.16",
+        "wait_command": true
+    }
+]
+```
+
+In this object, the `uid` is the database ID. You can use the database ID to view or update the database using the API.
+
+For more info on the fields returned by [GET `/v1/bdbs/`]({{<relref "/rs/references/rest-api/requests/bdbs/#get-all-bdbs" >}}), see the [`bdbs` object]({{<relref "/rs/references/rest-api/objects/bdb/" >}}).
+
+### PUT request
+
+Once you have the database ID, you can use [PUT `/v1/bdbs/`]({{<relref "/rs/references/rest-api/requests/bdbs/#put-bdbs" >}}) to update the configuration of the database.
+
+```sh
+$ curl -X PUT -H "accept: application/json" \
+            -H "content-type: application/json" \
+            -u "cameron.bates@redis.com:test123" \
+            https://[host]:[port]/v1/bdbs/1 \
+            -d '{ "name": "database1" }' -k -i
+HTTP/1.1 200 OK
+server: envoy
+date: Tue, 14 Jun 2022 20:00:25 GMT
+content-type: application/json
+content-length: 2933
+cluster-state-id: 43
+x-envoy-upstream-service-time: 159
+
+{
+    ...
+    "name" : "database1",
+    ...
+    "uid" : 1,
+    "version" : "6.0.16",
+    "wait_command" : true
+}
+```
+
+For more info on the fields required by [PUT `/v1/bdbs/`]({{<relref "/rs/references/rest-api/requests/bdbs/#put-bdbs" >}}), see the [`bdbs` object]({{<relref "/rs/references/rest-api/objects/bdb/" >}}).
+
+## More examples
+
```

Comment:
Maybe add a sentence that explains that you can also use client libraries to make requests with your preferred language.